packages/RNTester/js/examples/Alert/AlertExample.js

```diff
@@ -9,137 +9,169 @@
-      </View>
-    );
-  }
-}
-
-class AlertExample extends React.Component {
-  static title = 'Alert';
-
-  static description =
-    'Alerts display a concise and informative message ' +
-    'and prompt the user to make a decision.';
-
-  render() {
-    return (
-      <RNTesterBlock title={'Alert'}>
-        <SimpleAlertExampleBlock />
-      </RNTesterBlock>
-    );
-  }
-}
+            },
+            {text: 'No', onPress: () => console.log('No Pressed!')},
+            {text: 'Yes', onPress: () => console.log('Yes Pressed!')},
+          ])
+        }>
+        <View style={styles.button}>
+          <Text>Tap to view alert</Text>
+        </View>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
+const AlertWithManyButtons = () => {
+  const alertMessage =
+    'Credibly reintermediate next-generation potentialities after goal-oriented ' +
+    'catalysts for change. Dynamically revolutionize.';
+
+  return (
+    <View>
+      <TouchableHighlight
+        style={styles.wrapper}
+        onPress={() =>
+          Alert.alert(
+            'Foo Title',
+            alertMessage,
+            '..............'.split('').map((dot, index) => ({
+              text: 'Button ' + index,
+              onPress: () => console.log('Pressed ' + index),
+            })),
+          )
+        }>
+        <View style={styles.button}>
+          <Text>Tap to view alert</Text>
+        </View>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
+const AlertWithCancelableTrue = () => {
+  const alertMessage = 'Tapping outside this dialog will dismiss this alert.';
+
+  return (
+    <View>
+      <TouchableHighlight
+        style={styles.wrapper}
+        onPress={() =>
+          Alert.alert(
+            'Alert Title',
+            alertMessage,
+            [{text: 'OK', onPress: () => console.log('OK Pressed!')}],
+            {
+              cancelable: true,
+              onDismiss: () =>
+                console.log(
+                  'This alert was dismissed by tapping outside of the alert dialog.',
+                ),
+            },
+          )
+        }>
+        <View style={styles.button}>
+          <Text>Tap to view alert</Text>
+        </View>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
+const AlertWithStyles = () => {
+  const alertMessage = 'Look at the button styles!';
+
+  return (
+    <View>
+      <TouchableHighlight
+        style={styles.wrapper}
+        onPress={() =>
+          Alert.alert('Styled Buttons!', alertMessage, [
+            {
+              text: 'Default',
+              onPress: () => console.log('Default Pressed!'),
+              style: 'default',
+            },
+            {
+              text: 'Cancel',
+              onPress: () => console.log('Cancel Pressed!'),
+              style: 'cancel',
+            },
+            {
+              text: 'Destructive',
+              onPress: () => console.log('Destructive Pressed!'),
+              style: 'destructive',
+            },
```

Comment:
I'm not so sure about using console logs. When dealing with checking the validity of such options, it is often beneficial to include a `<Text>` whose value changes based on the option you click - refer to the `ActionSheetIOS` examples to see how this was done there. Doing this will also allow you to add additional e2e tests by leveraging the `expect(...)` and `.toHaveText()` ([look here for the latter](https://github.com/wix/Detox/blob/master/docs/APIRef.Expect.md#tohavetexttext)) functions. What are your thoughts on this?

Reply:
Thanks for the review Ansh!

I have pushed another commit which shows log on the screen itself.

<img width="307" alt="Screenshot 2020-07-11 at 10 58 24 AM" src="https://user-images.githubusercontent.com/22813027/87217317-73322200-c365-11ea-9337-4b751fce0d1f.png">

Reply:
Perfect! Exactly what I had in mind.